solvcon/parcel/gas/inout.py

```diff
@@ -65,12 +65,18 @@ def __init__(self, cse, show_bclist=False, perffn=None, **kw):
         super(MeshInfoHook, self).__init__(cse, **kw)
 
     def preloop(self):
+        flag_generic_bc = False
```

Comment:
Fixed. Thanks for the reminder. I scanned ``inout.py`` this time by the code checker ``flake8``. I fixed all coding style issues related to this PR. The other pep8 issues prompted by ``flake8`` are not fixed to make the code review be easier. They can be fixed later in another PR.